app/src/main/java/a/a/a/yq.java

```diff
@@ -840,29 +833,52 @@ public void b(hC hCVar, eC eCVar, iC iCVar, boolean z2) {
     }
 
     /**
-     * Get source code files that are viewable in SrcCodeViewer (?)
+     * Get source code files that are viewable in SrcCodeViewer
      */
     public ArrayList<SrcCodeBean> a(hC hCVar, eC eCVar, iC iCVar, boolean z2) {
         a(iCVar, hCVar, eCVar, z2);
         CommandBlock.x();
+        
+        final String javaDir = FileUtil.getExternalStorageDir() + "/.sketchware/data/" + d + "/files/java/";
+        final String layoutDir = FileUtil.getExternalStorageDir() + "/.sketchware/data/" + d + "/files/resource/layout/";
```

Comment:
I've just checked the Javadocs for those fields in yq, and they apparently have the project name instead of the sc_id, is that right?

Reply:
you're right, it should be the sc_id 👀

Reply:
anyway, that's not too important for now. great commit so far!